src/mapblock_mesh.cpp

```diff
@@ -1040,6 +1042,32 @@ MapBlockMesh::MapBlockMesh(MeshMakeData *data, v3s16 camera_offset):
 {
 	m_enable_shaders = data->m_use_shaders;
 	m_enable_highlighting = g_settings->getBool("enable_node_highlighting");
+	
+	if (g_settings->getBool("enable_minimap")) {
+		v3s16 blockpos_nodes = data->m_blockpos * MAP_BLOCKSIZE;
+		for(s16 x = 0; x < MAP_BLOCKSIZE; x++) {
+			for(s16 z = 0; z < MAP_BLOCKSIZE; z++) {
+				s16 air_count = 0;
+				bool surface_found = false;
+				MinimapPixel* minimap_pixel = &m_minimap_mapblock->data[x + z * MAP_BLOCKSIZE]; 
+				for(s16 y = MAP_BLOCKSIZE -1; y > -1; y--) {
+					v3s16 p(x, y, z);
+					MapNode n = data->m_vmanip.getNodeNoEx(blockpos_nodes + p);
+					if (!surface_found && n.param0 != CONTENT_AIR) {
+						minimap_pixel->height = y;
+						minimap_pixel->id = n.param0;
+						surface_found = true;
+					} else if (n.param0 == CONTENT_AIR) {
+						air_count++;
+					}
+				}
+				if (!surface_found) {
```

Comment:
I know the code style permits one-line control statements to have brackets or no brackets, but you keep changing between the two.  Please, pick one style and stick with it!